tests/integration_tests/data_steward/validation/main_test.py

```diff
@@ -58,6 +57,7 @@ def setUp(self):
         self.storage_client.empty_bucket(self.drc_bucket)
         self.storage_client.empty_bucket(self.hpo_bucket)
 
+        test_util.delete_all_tables(self.bq_client, self.dataset_id)
```

Comment:
This line is identical to the line below. If my understanding is correct, we want it for `self.rdr_dataset_id`.